mythril/analysis/security.py

```diff
@@ -4,19 +4,29 @@
 import logging
 
 
-def fire_lasers(statespace, module_names=None):
-
-    issues = []
+def get_detection_modules(entrypoint, except_modules=None):
+    except_modules = [] if except_modules is None else except_modules
+    except_modules.append("base")  # always exclude base class file
```

Comment:
```suggestion
    except_modules = ["base"].extend(except_modules)
```

Reply:
and make except modules have the default ```()```

Reply:
`list.extend` returns `None` and changes the list in place. So that wouldn't have any effect. But if you want to keep the default argument an iterable, we could do something like
```python
def get_detection_modules(entrypoint, except_modules=()):
    except_modules = list(except_modules) + ["base"]
```
That way we always have a list with at least `base` in it to call all familiar methods on, while the default argument stays immutable (which is *really* important. :smile: )